examples/resources/pingfederate_captcha_provider/resource.tf

```diff
@@ -0,0 +1,22 @@
+resource "pingfederate_captcha_providers" "captchaProviderExample" {
+  providers_id = "myCaptchaProviderId"
```

Comment:
This attribute should be singular as well

Reply:
done